contracts/Storage/ChallengesStore.sol

```diff
@@ -83,14 +83,11 @@ contract ChallengesStore is Base {
         return keccak256(abi.encodePacked(sheltererId, bundleId));
     }
 
-    function increaseSequenceNumber(bytes32 challengeId) public onlyContextInternalCalls {
-        challenges[challengeId].sequenceNumber++;
-    }
-
     function decreaseActiveCount(bytes32 challengeId) public onlyContextInternalCalls {
         activeChallengesOnBundleCount[challenges[challengeId].bundleId] = activeChallengesOnBundleCount[
             challenges[challengeId].bundleId].sub(1).castTo32();
         challenges[challengeId].activeCount = challenges[challengeId].activeCount.sub(1).castTo8();
+        increaseSequenceNumber(challengeId);
```

Comment:
As this is the only place the `increaseSequenceNumber` private method is called I would move the one line it contains into this method. Otherwise this will be inconsitant with the above line where we modify a field directly.